Doberman/AlarmMonitor.py

```diff
@@ -121,47 +121,43 @@ def send_email(self, toaddr, subject, message, cc=None, bcc=None, add_signature=
+        connection_details = self.get_connection_details('smscreator')
         if connection_details is None:
-            return -1
+            raise KeyError("No connection details obtained from database.")
         # Compose connection details and addresses
-        try:
-            server = connection_details['server']
-            identification = connection_details['identification']
-            contactaddr = connection_details['contactaddr']
-            # fromaddr = connection_details['fromaddr']
-            if not phone_number:
-                self.logger.warning('No phone number given. Can not send SMS.')
-                return 0
-            # Server has different type request for 1 or several numbers.
-            if len(phone_number) == 1:
-                toaddr = f'{identification}.{phone_number[0]}@{server}'
-                bcc = None
-            else:
-                toaddr = contactaddr
-                bcc = [f'{identification}.{number}@{server}' for number in phone_number]
-            message = str(message)
-            subject = ''
-            # Long SMS (>160 characters) cost more and are shortened
-            if len(message) > 155:
-                self.logger.warning('SMS message exceeds limit of 160 characters '
-                                    f'({len(message)} characters). '
-                                    'Message will be cut off.')
-                message = message[:155]
-            cc = None
-            if self.send_email(toaddr=toaddr,
-                               subject=subject,
-                               message=message,
-                               cc=cc, bcc=bcc,
-                               add_signature=False) == -1:
-                self.logger.error('Could not send SMS! Email to SMS not working.')
-                return -1
+        url = connection_details['url']
+        postparameters = connection_details['postparameters']
+        maxmessagelength = int(connection_details['maxmessagelength'])
+        auth = connection_details.get('auth')
+        if not phone_numbers:
+            raise ValueError("No phone number given.")
+
+        message = str(message)
+        # Long messages are shortened to avoid excessive fees
+        if len(message) > maxmessagelength:
+            message = ' '.join(message[:maxmessagelength + 1].split(' ')[0:-1])
+            self.logger.warning(f"Message exceeds {maxmessagelength} "
+                                "characters. Message will be shortened.")
+
+        if type(phone_numbers) == str:
+            phone_numbers = [phone_numbers]
+        for tonumber in phone_numbers:
+            data = postparameters
+            data['Recipient'] = tonumber
+            data['SMSText'] = message
+            self.logger.warning(f'Sending SMS to {tonumber}')
+            print(data)
+            response = requests.post(url, data=data)
+            print(response.status_code)
```

Comment:
These two print statements should either get removed or changed to log calls

Reply:
They're gone